### PR TITLE
Adding known issues for policy templates

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/stolostron/backlog/issues/27150

### CODE REVIEW

**Comment by dockerymick** [comment]
Assigning @mprahl because he created the issue. Let me know if I should reassign

File: release_notes/known_issues.adoc [hunk 1]
```diff
@@ -573,6 +573,16 @@ To resolve the issue, disable the policy and compare the differences between `ob
 +
 To resolve the issue, disable the policy and compare the differences between `objectDefinition` in the policy and the object on the managed cluster. If the keys are different or missing, Kubernetes might have processed the keys before applying them to the object. For `PodSecurityPolicy`, for example, Kubernetes removes keys with values set to `false`, which you can see in the resulting object on the managed cluster. In this case, remove the keys from the `objectDefinition` in the policy.
 
+
+[#policy-templates-not-removed]
+=== Policy template issues
+
+- When you rename your configuration policy to a new name, a copy of the configuration policy with the older name remains.
```
Comment (oafischer) [inline release_notes/known_issues.adoc:603]: Do we want to start with a list right away? Might be better to have an intro paragraph or just a short prompt before that.

**Review by oafischer — APPROVED** [review]
Looks good, one suggestion but I'll leave it up to you.

File: release_notes/known_issues.adoc (hunk 1 above)
Comment (dockerymick) [reply release_notes/known_issues.adoc:603]: Thanks Oliver! I updated the file

File: release_notes/known_issues.adoc [hunk 2]
```diff
@@ -573,6 +573,17 @@ To resolve the issue, disable the policy and compare the differences between `ob
...
 To resolve the issue, disable the policy and compare the differences between `objectDefinition` in the policy and the object on the managed cluster. If the keys are different or missing, Kubernetes might have processed the keys before applying them to the object. For `PodSecurityPolicy`, for example, Kubernetes removes keys with values set to `false`, which you can see in the resulting object on the managed cluster. In this case, remove the keys from the `objectDefinition` in the policy.
 
+[#policy-templates-not-removed]
+=== Policy template issues
+
+You might encounter the following issues when you edit policy templates for configuration policies:
+
+- When you rename your configuration policy to a new name, a copy of the configuration policy with the older name remains. 
+
+- If you remove a configuration policy from your hub cluster, the configuration policy remains on your managed cluster. A status is not provided.
```
Comment (dhaiducek) [inline release_notes/known_issues.adoc]:
```suggestion
- If you remove a configuration policy from a policy on your hub cluster, the configuration policy remains on your managed cluster but its status is not provided.
```

File: release_notes/known_issues.adoc [hunk 3]
```diff
@@ -573,6 +573,17 @@ To resolve the issue, disable the policy and compare the differences between `ob
...
+
+To resolve this, disable your configuration and reenable it. You can also delete the entire policy.
```
Comment (dhaiducek) [inline release_notes/known_issues.adoc]:
```suggestion
To resolve this, disable your policy and reenable it. You can also delete the entire policy.
```

**Review by dhaiducek — CHANGES_REQUESTED** [review]
Minor suggestions

**Comment by dockerymick** [comment]
@dhaiducek thanks so much Dale! I have accepted your suggestions.

**Review by dhaiducek — APPROVED** [review]

**Comment by dockerymick** [comment]
Thanks for the approvals! @dhaiducek @oafischer 

Merging this PR

**Pull request merged**